.github/workflows/on-push-workflow.yml

```diff
@@ -94,7 +96,7 @@ jobs:
             BixiLocationCollection=$BIXI_LOCATION_COLLECTION \
             BixiTripCollection=$BIXI_TRIP_COLLECTION \
             BixiChunkSize=$BIXI_CHUNK_SIZE \
-            UpdateCheck="${BIXI},${STMAnalyzeDailyStop},${STMCreateDailyStop},${STMFetchTripUpdates},${STMFetchVehiclePositions},${STMFetchUpdateStatic},${STMFilterDailyStatic},${STMMergeDailyVehiclePosition},${BixiHistoricDataChecker},${BixiHistoricDataProcessor}"
+            UpdateCheck="${BIXI},${STMAnalyzeDailyStop},${STMCreateDailyStop},${STMFetchTripUpdates},${STMAnalyzeSegments},${STMFetchVehiclePositions},${STMFetchUpdateStatic},${STMFilterDailyStatic},${STMMergeDailyVehiclePosition},${BixiHistoricDataChecker},${BixiHistoricDataProcessor}"
```

Comment:
Position doesn't match with the condition in the template.
To keep it consistent, STMAnalyzeSegments should be at the end of this list

Reply:
Right, thanks for pointing this out I overlooked this, fixed